templates/index.html

```diff
@@ -1,7 +1,14 @@
 {% extends "base.html" %}
 
 {% block content %}
-Hello World!!!
+	<div class="inner cover">
```

Comment:
This isn't the header, it's the center of the homepage. I added the header separately in base :)